src/components/App.tsx

```diff
@@ -26,6 +34,46 @@ export function App(): ReactElement {
           </Route>
         </Switch>
       </div>
+      <nav className="navbar fixed-bottom navbar-light bg-light">
+        <a className="navbar-link" href="https://github.com/inferno-community/fhir-validator-app">
```

Comment:
should we give these links a `target="_blank"` attribute, so they open in new tabs?